src/components/ui/Pressable/Pressable.stories.tsx

```diff
@@ -0,0 +1,27 @@
+import type { ComponentMeta, ComponentStory } from '@storybook/react';
+import Pressable from './Pressable';
+
+export default {
+  title: 'Ui/Pressable',
+  component: Pressable,
+  argTypes: {},
+} as ComponentMeta<typeof Pressable>;
+
+const Template: ComponentStory<typeof Pressable> = (args) => (
+  <Pressable {...args}>
+    <p>Click me</p>
```

Comment:
Let's keep it as plain text and remove `p` from here, so user is able to see how it looks with plain text inside